azure-go-sdk-conceptual/index.yml

```diff
@@ -0,0 +1,79 @@
+### YamlMime:YamlDocument
+documentType: LandingData
+title: Azure for Go developers
+metadata:
+  title: Azure for Go developers - Tutorials, API Reference
+  description: Get started developing with the Azure SDK for Go using the developer resources below.
+  keywords: Azure, Go, golang, SDK, API
+  author: sptramer
+  ms.author: sttramer
+  manager: routlaw
+  ms.date: 01/30/2018
+  ms.topic: article
+  featureFlags:
+  - clicktale
+  ms.prod: azure
+  ms.technology: azure
+  ms.devlang: golang
+  ms.service: multiple
+sections:
+- items:
+  - type: markdown
+    text: |
+      Get started developing apps with the Azure SDK for Go using our official tutorials and tools.
+- items:
+  - type: list
+    style: cards
+    className: cardsM
+    columns: 2
+    items:
+    - title: Get Started
+      html: <p>Install the Azure SDK for Go</p>
+      image:
+        src: https://docs.microsoft.com/media/common/i_get-started.svg
+    - title: Code Samples
+      html: <p>Azure code samples using Go</p>
+      image:
+        src: https://docs.microsoft.com//media/common/i_code-samples.svg
+      href: https://github.com/azure-samples/azure-sdk-for-go-samples
+    - title: SDK &amp; Tools
+      html: <p>Developer tools</p>
+      href: https://docs.microsoft.com/go/azure/azure-sdk-go-tools 
+      image:
+        src: https://docs.microsoft.com/media/common/i_download-install.svg
+    - title: API Reference
+      html: <p>GoDoc for Azure SDK for Go</p>
+      href: https://godoc.org/github.com/Azure/azure-sdk-for-go
+      image:
+        src: https://docs.microsoft.com/media/common/i_api-reference.svg
+- title: Quickstarts
+  items:
+  - type: paragraph
+    text: 'Learn how to work with Azure services from Go.'
+  - type: list
+    style: cards
+    className: cardsM
+    columns: 2
+    items: 
+    - title: Deploy a virtual machine to Azure

```

Comment:
Recommend changing to "Use the Go SDK to deploy an Azure VM" – it might be confusing otherwise, with people expecting a guide on how to run Go in Azure VMs (it confused me too...)

Reply:
Considering this tile is located directly underneath the header "Learn how to work with Azure services from Go" I think there is enough context.

Reply:
I disagree with that. It confused me when I looked at the page, and then I opened the article and couldn't figure out what it was about.